### PR TITLE
tag retry

### PR DESCRIPTION
**Pull request by sahilvirani**
Okay, I was able to redo it. The files changes are the communityservice, communitydto, community, community controller.

### CODE REVIEW

**Review by shreya-devaraju — APPROVED** [review]
If it looks good to Roham it looks good to me! Thank you so much Sahil!!

File: src/main/java/iluvus/backend/api/service/CommunityService.java
Comment (shreya-devaraju) [inline src/main/java/iluvus/backend/api/service/CommunityService.java:1]: Why did you change this part if (moderators != null || !moderators.isBlank()) to be this : if (moderators != null && !moderators.isBlank()) ?

Comment (sahilvirani) [reply src/main/java/iluvus/backend/api/service/CommunityService.java:1]: Oh, I didn't mean to do that. I was thinking about what it should be and wanted to ask you guys about it first.

**Review by DevikaPapal — APPROVED** [review]
thanks!
